docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -158,16 +158,16 @@ You can validate the updater is running properly by checking if its pod is ready
 ```code
 $ kubectl get pods
 NAME                               READY   STATUS    RESTARTS   AGE
-my-agent-0                         1/1     Running   0          14m
-my-agent-1                         1/1     Running   0          14m
-my-agent-2                         1/1     Running   0          14m
-my-agent-updater-d9f97f5dd-v57g9   1/1     Running   0          16m
+<your-agent-release>-0                         1/1     Running   0          14m
+<your-agent-release>-1                         1/1     Running   0          14m
+<your-agent-release>-2                         1/1     Running   0          14m
+<your-agent-release>-updater-d9f97f5dd-v57g9   1/1     Running   0          16m
 ```
 
 And by consulting its logs:
 
 ```code
-$ kubectl logs <your-agent-release>-updater
+$ kubectl logs deployment/<your-agent-release>-updater
```

Comment:
Is it a deployment or a statefulset? Looks like the latter to me.

Reply:
The updater is a deployment. The agent itself is a statefulset 
https://github.com/gravitational/teleport/blob/master/examples/chart/teleport-kube-agent/templates/updater/deployment.yaml